cli/ceph/balancer.py

```diff
@@ -16,3 +16,83 @@ def status(self):
         if isinstance(out, tuple):
             return out[0].strip()
         return out
+
+    def set_state(self, state):
+        """
+        Sets balancer state to ON/OFF
+        Args:
+            state (str): state of balancer on/off
+        """
+        cmd = f"{self.base_cmd} {state}"
+        out = self.execute(sudo=True, check_ec=False, long_running=False, cmd=cmd)
+        if isinstance(out, tuple):
+            return out[0].strip()
+        return out
+
+    def mode(self, mode):
+        """
+        Sets the balancer mode
+        Args:
+            mode (str): mode to be set (crush-compat / upmap)
+        """
+        cmd = f"{self.base_cmd} mode {mode}"
+        out = self.execute(sudo=True, check_ec=False, long_running=False, cmd=cmd)
+        if isinstance(out, tuple):
+            return out[0].strip()
+        return out
+
+    def eval(self, plan=None, pool_name=None, verbose=False):
+        """
+        Performs evaluation and score the current distribution
+        Args:
+            plan (str): Plan to be evaluated
+            pool_name (str): Evaluate the distribution for a single pool
+            verbose (bool): Set True to see greater detail for the evaluation
+        """
+        cmd = f"{self.base_cmd} eval"
+        if verbose:
+            cmd += "-verbose"
+        if pool_name:
+            cmd += f" {pool_name}"
+        if plan:
+            cmd += f" {plan}"
+        out = self.execute(sudo=True, check_ec=False, long_running=False, cmd=cmd)
```

Comment:
Can we remove "check_ec=False, long_running=False" since by defult it is false and get parmeter from kw

https://github.com/red-hat-storage/cephci/blob/master/cli/__init__.py#L5

Example:https://github.com/red-hat-storage/cephci/blob/master/cli/ceph/mgr.py#L27